src/renderers/shaders/ShaderChunk/normal_fragment_begin.glsl.js

```diff
@@ -19,7 +19,7 @@ float faceDirection = gl_FrontFacing ? 1.0 : - 1.0;
 
 #endif
 
-#ifdef USE_NORMALMAP_TANGENTSPACE
+#if defined( USE_NORMALMAP_TANGENTSPACE ) || defined( USE_CLEARCOAT_NORMALMAP ) || defined( USE_ANISOTROPY )
```

Comment:
@elalish It looks like there is an issue with this particular line of code. Users report about assets that throw a shader error now:
```
Program Info Log: Fragment shader is not compiled.


FRAGMENT

ERROR: 0:1384: 'vNormalMapUv' : undeclared identifier
ERROR: 0:1384: 'getTangentFrame' : no matching overloaded function found
ERROR: 0:1384: '=' : dimension mismatch
ERROR: 0:1384: '=' : cannot convert from 'const mediump float' to 'highp 3X3 matrix of float'


  1379: #endif
  1380: #if defined( USE_NORMALMAP_TANGENTSPACE ) || defined( USE_CLEARCOAT_NORMALMAP ) || defined( USE_ANISOTROPY )
  1381: 	#ifdef USE_TANGENT
  1382: 		mat3 tbn = mat3( normalize( vTangent ), normalize( vBitangent ), normal );
  1383: 	#else
> 1384: 		mat3 tbn = getTangentFrame( - vViewPosition, normal, vNormalMapUv );
  1385: 	#endif
  1386: 	#if defined( DOUBLE_SIDED ) && ! defined( FLAT_SHADED )
  1387: 		tbn[0] *= faceDirection;
  1388: 		tbn[1] *= faceDirection;
  1389: 	#endif
  1390: #endif
```
It seems not right to use `USE_CLEARCOAT_NORMALMAP` at this point since `vNormalMapUv` is _only_ define when a normal map is present (meaning when `USE_NORMALMAP` is defined).

https://discourse.threejs.org/t/shader-error-0-validate-status-false/53195

Reply:
Ah, I may have missed that case. I did add [this](https://github.com/mrdoob/three.js/pull/25580/files#diff-fc94afb5da943fea9e01a4a16adf82b5433aa6d61af5855f296158a98ed4dcfdR1615) to ensure the tangents were defined, but perhaps a similar thing is necessary for the UVs? Anisotropy requires UV coordinates (and ideally tangents too), even if no normal map is present. I think it's perfectly reasonable to throw an error if anisotropy is requested without any UV channels being present (that's in the glTF extension spec too).

Reply:
Hey sorry to jump in, I was debugging this bug and got here. 
@elalish It's possible to have UVs without a normal map being present, which won't create the vNormalMapUv in the shader. 
I solved it by setting USE_UV in material.defines that will make vUv available and use that when normal map is not available.

Reply:
Yes, I think that's the solution - to set `USE_UV` in the WebGLRenderer when anisotropy is present even if there's no normal map. Want to try a PR? I'll be happy to review.

Reply:
It's also necessary to support the clear coat case. Otherwise the car asset will still fail (it does not use anisotropy).

Reply:
Besides, I don't think `USE_UV` solve the above issue. `USE_NORMALMAP` defines the `vNormalMapUv` varying which is required to make the shader code work.

Reply:
Yeah, it would be nice if the UVs were a little less closely-tied to particular maps, but I'm not sure how much surgery you want to do, @Mugen87. Basically we just need a UV coordinate when anisotropy is used, even if maps aren't present. I think there is some tension between the smallest change to make that work, and the most readable.

Reply:
I'm okay with a hot fix (smallest change) for now. Things can be refactored later.

Reply:
I created a small PR https://github.com/mrdoob/three.js/pull/26334 With these changes the model from sketchfab loads without throwing an error.